### DISCUSSION TITLE
[Proposal] PagerControl

### DISCUSSION THREAD

**Discussion by jingliancui** [CommunityToolkit/Maui#1629] [New Feature Discussions]
### Feature name

PagerControl

### Link to discussion

https://github.com/dotnet/maui/issues/19615

### Progress tracker

- [ ] Android Implementation
- [ ] iOS Implementation
- [ ] MacCatalyst Implementation
- [ ] Windows Implementation
- [ ] Tizen Implementation
- [ ] Unit Tests
- [ ] Samples
- [ ] Documentation

### Summary

https://github.com/microsoft/microsoft-ui-xaml/issues/60

### Motivation

https://github.com/microsoft/microsoft-ui-xaml/issues/60

### Detailed Design

https://github.com/microsoft/microsoft-ui-xaml/issues/60

### Usage Syntax

```markdown
https://github.com/microsoft/microsoft-ui-xaml/issues/60
```

**Comment by jingliancui** [comment]
also open request on https://github.com/enisn/UraniumUI/issues/525 and https://github.com/dotnet/maui/issues/19615

**Comment by CliffAgius** [comment]
Closed as per the Community Stand-up discussion here: https://youtu.be/N9wMcBP4jtg?t=2889 where it was agreed that New Features need to get above 7 upvotes within 6 months to progress to being reviewed by the maintainers. 

If you feel this is being closed in error please let us know and we can re-open.
